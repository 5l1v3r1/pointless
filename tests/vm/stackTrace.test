
output = foo(z) |> println
foo(z) = z.y
z = {"x": 123}
