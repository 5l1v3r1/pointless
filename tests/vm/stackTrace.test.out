-------------------------------------------------------------------------------
Type Error:

Field ref expected type 'PtlsObject', got type 'PtlsMap'
-------------------------------------------------------------------------------

At (line 3) (column 11) in 'tests/vm/stackTrace.test'
foo(z) = z.y
          ^
At (line 2) (column 20) in 'tests/vm/stackTrace.test'
output = foo(z) |> println
                   ^
